wordpress-mobile/WordPress-iOS
# UL&S: SiteCredentialsViewController: username and password fields

Ref. #14009 
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/329/

### To test
1. Enable the `.siteAddress` feature flag
2. `rake dependencies`
3. Build and run
4. Log in > Or log in by entering your site address > enter a valid URL > Continue
5. Test out the username and password fields. The password field should toggle between visible and hidden states. The cursor should be blue while the eye icon should be gray. The username field keyboard should say "next" and move the cursor from the username field to the password field. The password field keyboard should say "continue". The Continue button implementation is not part of this PR.

### Screenshots
| iPhone 11 Pro | Dark Mode |
| --- | --- |
| <a href="https://user-images.githubusercontent.com/1062444/87967437-68be0980-ca84-11ea-9913-ffb3858206eb.png"><img src="https://user-images.githubusercontent.com/1062444/87967437-68be0980-ca84-11ea-9913-ffb3858206eb.png" width="350" /></a> | <a href="https://user-images.githubusercontent.com/1062444/87967442-6bb8fa00-ca84-11ea-83d9-0139d13fa2f6.png"><img src="https://user-images.githubusercontent.com/1062444/87967442-6bb8fa00-ca84-11ea-83d9-0139d13fa2f6.png" width="350" /></a>
| <a href="https://user-images.githubusercontent.com/1062444/87967452-707dae00-ca84-11ea-8be0-47f17e0e7a77.png"><img src="https://user-images.githubusercontent.com/1062444/87967452-707dae00-ca84-11ea-8be0-47f17e0e7a77.png" width="350" /></a> |

PR submission checklist:
